examples/object-closure/object_closure.bal

```diff
@@ -0,0 +1,19 @@
+import ballerina/io;
+
+public function main() {
+    string[] names = ["Ana", "Alice", "Bob"];
+
+    // Create an object value using an object constructor.
+    var engineer = object {
+        string name = "";
+
+        function setName(string name) {
+            // Access `names` variable as a closure within the `setName` method in the object constructor.
+            names.push(name);
+            self.name = name;
+        }
+    };
+
+    engineer.setName("Walter");
```

Comment:
Since we are setting the passed name, shall we print the `engineer.name` too?

Reply:
I agree. I will update